docs/tools/hedgehog-engine/rangers/files/material.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
This file is not referenced anywhere in the `mkdocs.yml` file. Please add it so that it gets recognized.